src/lxc/lxccontainer.h

```diff
@@ -496,7 +496,7 @@ struct lxc_container {
 	bool (*set_cgroup_item)(struct lxc_container *c, const char *subsys, const char *value);
 
 	/*!
-	 * \brief Determine full path to the containers configuration file.
+	 * \brief Determine full path to the containers configuration directory.
```

Comment:
Don't want to be nitpicky, but could you maybe add also an example (like `e.g. /var/lib/lxc`)?

Reply:
I believe the paragraph below mentioning it uses the LXCPATH build time variable is sufficient. Providing an exact path when it's dependent on how it's built and shipped doesn't help all that much.